specification/paths/traffic/trafficRoute.yml

```diff
@@ -0,0 +1,42 @@
+servers:
+  - url: https://api.woosmap.com
+tags:
+  - Traffic API
+description: >
+  Get distance, duration and path (as a polyline) for a pair of origin and destination, based on the recommended route between those two points (including traffic).
+security:
+  - PublicApiKeyAuth: []
+    RefererHeader: []
+  - PrivateApiKeyAuth: []
+parameters:
+  - $ref: "../../parameters/distance/origin.yml"
```

Comment:
In the origin description, there is a reference to isochrone that should not exist.